From 8af970fd687eeeeda166c2961821d9e085dbd67b Mon Sep 17 00:00:00 2001
From: Moritz Horstmann <dev@peterzweg.at>
Date: Sun, 08 Apr 2018 02:04:19 +0200
Subject: [PATCH] manifest: Track specific revision of svox to fix security vulnerability

Unfortunately, github does not allow us to fork svox, so we'll use
the particular commit ID which fixes CVE-2017-13285 RCE vulnerability.
Even though this revision originates from Oreo, the difference to the
current nougat branch is negligible.

Change-Id: Ib0e6e7c7be8ff60584e1c6cb5fb677d87e1dbc12
---

diff --git a/default.xml b/default.xml
index 4402780..cfb7f17 100644
--- a/default.xml
+++ b/default.xml
@@ -273,7 +273,7 @@
   <project path="external/squashfs-tools" name="platform/external/squashfs-tools" groups="pdk" remote="aosp" />
   <project path="external/srtp" name="platform/external/srtp" groups="pdk" remote="aosp" />
   <project path="external/strace" name="platform/external/strace" groups="pdk" remote="aosp" />
-  <project path="external/svox" name="platform/external/svox" groups="pdk" remote="aosp" revision="refs/tags/android-n-mr2-preview-2" />
+  <project path="external/svox" name="platform/external/svox" groups="pdk" remote="aosp" revision="cee78199bbfae81f54a40671db47096f5f32cdad" />
   <project path="external/tagsoup" name="platform/external/tagsoup" groups="pdk" remote="aosp" />
   <project path="external/testng" name="platform/external/testng" groups="pdk" remote="aosp" />
   <project path="external/tcpdump" name="platform/external/tcpdump" groups="pdk" remote="aosp" />
